.gitignore

```diff
@@ -12,8 +12,8 @@ venv/
 *.json
 *.csv
 *.zip
-*.xls
-*.xlsx
+/*.xls
+/*.xlsx
```

Comment:
Ao  invés de colocar dessa forma, acho que podemos manter o padrão anterior e abrir [uma exceção](https://stackoverflow.com/questions/2415873/exceptions-in-gitignore/20731396) para os `xls` do que estiver em pastas de testes. O que acha?
(não tive como testar ainda)

Reply:
Testei aqui e funcionaria:

```
*.xls
*.xlsx

# arquivos de testes
!**/tests/**/**.xls
!**/tests/**/**.xlsx
```

Uma outra opção seria `!**/tests/files/**.xls`.
A primeira opção funcionaria para pastas e subpastas dentro de `tests` e a segunda apenas para uma pasta `files` dentro de `tests`.